api/daemon.go

```diff
@@ -114,6 +114,7 @@ func fetchLatestRelease() (githubRelease, error) {
 	if err != nil {
 		return githubRelease{}, err
 	}
+	fmt.Printf("[FOR TESTING]\nrelease: %v\n", release)
```

Comment:
see the Travis output for Mac vs. Linux: 

Travis Darwin: 
`release: { []}`


Travis Linux & my Mac:
`release: {v1.0.0 [{Sia-v1.0.0-darwin-amd64.zip https://github.com/NebulousLabs/Sia/releases/download/v1.0.0/Sia-v1.0.0-darwin-amd64.zip} {Sia-v1.0.0-linux-amd64.tar.gz https://github.com/NebulousLabs/Sia/releases/download/v1.0.0/Sia-v1.0.0-linux-amd64.tar.gz} {Sia-v1.0.0-windows-amd64.zip https://github.com/NebulousLabs/Sia/releases/download/v1.0.0/Sia-v1.0.0-windows-amd64.zip}]}`